src/test/java/uk/gov/pay/adminusers/persistence/dao/UserDaoTest.java

```diff
@@ -177,7 +178,7 @@ public void shouldFindUsersBy_ExternalIds() throws Exception {
         List<String> externalIds = Arrays.asList(user1.getExternalId(), user2.getExternalId());
 
         List<UserEntity> userEntities = userDao.findByExternalIds(externalIds);
-        assertTrue(userEntities.size() == 2);
+        assertEquals(2, userEntities.size());
```

Comment:
`assertThat(userEntities.size(), is(2));` would be more in line with our usual conventions

Reply:
Addressed in https://github.com/alphagov/pay-adminusers/pull/350